app/src/main/java/net/squanchy/about/licenses/Libraries.kt

```diff
@@ -3,32 +3,33 @@ package net.squanchy.about.licenses
 internal object Libraries {
 
     val LIBRARIES = listOf(
-            Library("Squanchy", "the Squanchy Authors", License.APACHE_2),
```

Comment:
This bunch of changes is just me reindenting things. The only addition is RxLint.